src/main/java/bio/terra/cda/app/service/QueryService.java

```diff
@@ -223,6 +228,10 @@ private void logQuery(Job queryJob, List<JsonNode> jsonData) {
     }
   }
 
+  /**
+   * @param query
```

Comment:
It's good to add javadoc but an empty javadoc should be deleted as it doesn't add anything to the code.

Reply:
fixed

Reply:
Marked as fixed but I don't see any changes in the PR diff

Reply:
There was some commented out code that somehow got back in.   I removed it again.

Reply:
This is still here, but empty. Please either delete or add javadoc to the comment.